it.baeyens.arduino.common/src/it/baeyens/arduino/arduino/Serial.java

```diff
@@ -98,6 +98,9 @@ static public void errorMessage(String where, Throwable e) {
 
     int stopbits;
     boolean monitor = false;
+    
+    // whether to pull DTR to GND during reset or not
```

Comment:
This comment is wrong. As to the jssc doc It should state "initial state of RTS&DTR line(ON/OFF)"
I would also add: Some boards reset when the serial port is opened with RTS and DTR low.
Basically the value is not used during reset so it is confusing to reference reset.
jssc doc I reference to http://java-simple-serial-connector.googlecode.com/svn/trunk/additional_content/javadoc/0.8/index.html